guides/source/action_mailer_basics.md

```diff
@@ -422,6 +422,27 @@ use the rendered text for the text part. The render command is the same one used
 inside of Action Controller, so you can use all the same options, such as
 `:text`, `:inline` etc.
 
+In the case that you have a Rails application with an unorthodox file organization and wish to render a template located outside of the default `app/views/mailer_name/` directory, you can apply the `prepend_view_path`, like so:
+
+```ruby
+class UserMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def welcome_email
+    @user = params[:user]
+    full_template_path = Rails.root.to_s + "/custom_folder/user_mailer/views/"
+    prepend_view_path(full_template_path)
```

Comment:
We usually recommend this method to be at class level.

Reply:
You mean you'd recommend doing it like so?

```

class UserMailer < ApplicationMailer
  default from: 'notifications@example.com'
  # Declaring it here to add it to the available paths
  full_template_path = Rails.root.to_s + "/custom_folder/user_mailer/views/"
  prepend_view_path(full_template_path)

  def welcome_email
    # Implementation
  end
end
```

Reply:
If so, I can redo this and add it like that, and note that its recommended to call it at the class level.

Reply:
Yes, that is it. You don't need to add a note, but this method is called for the entire controller. You only need to call it at the instance if it only make sense to only one action.

Reply:
So in the example, it would make most sense to call it at the class level?